### PR TITLE
Addding relocation type for PPC 32 EABI

### PR DESCRIPTION
**Pull request by rajez133**

### CODE REVIEW

File: elftools/elf/descriptions.py [hunk 1]
```diff
@@ -157,6 +157,8 @@ def describe_reloc_type(x, elffile):
         return _DESCR_RELOC_TYPE_MIPS.get(x, _unknown)
     elif arch == 'LoongArch':
         return _DESCR_RELOC_TYPE_LOONGARCH.get(x, _unknown)
+    elif arch == 'PowerPC':
```
Comment (eliben) [inline elftools/elf/descriptions.py:160]: Place this next to the other ppc line a bit above

**Review by eliben — CHANGES_REQUESTED** [review]
Thanks, just a minor request

**Comment by sevaa** [comment]
@rajez133 are you still interested in this?

**Comment by sevaa** [comment]
@eliben Requested changes were provided, I think.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,23 +2,22 @@ name: pyelftools-tests
 on:
   push:
     branches:
-      - master
+      - main
```
Comment (eliben) [inline .github/workflows/ci.yml:5]: Please sync this PR with the `main` branch, this should already be updated

Comment (rajez133) [reply .github/workflows/ci.yml]: Done.

**Comment by sevaa** [comment]
@eliben required changes were provided, no?

**Comment by eliben** [comment]
@sevaa 
Yes

this LGTY?

**Comment by sevaa** [comment]
I didn't check the changes against the ABI. and I don't have a binary to test this against, so I'd take the OP's word on it. The code looks in line with the rest of the library.

**Review by eliben — APPROVED** [review]

**Pull request merged**